### DISCUSSION TITLE
Share components from Igor-Lib?

### DISCUSSION THREAD

**Discussion by wayland** [shanteacontrols/OpenDeck#192] [Q&A]
Hi!  

I've been doing some more work on my modular idea.  Most of the components in the schematic were available in the standard KiCAD libraries, but there are a few that don't seem to be there.  Are you able to share the following from the Igor libraries (or alternately, tell me where I get these Igor libraries?)

- RP2040
- 74HC138 (I was surprised this wasn't available
- Crystal-4P
- TYPE-C-31-M-12
- RFM-0505
- TLP2362

Thanks!

**Comment by paradajz** [comment]
Which design are you talking about and which KiCAD version do you use? The footprints are embedded in the board file. I've just tried to open OpenDeck-r3.1.0 in virtual machine with clean install of KiCAD and I've loaded one of the footprints from my library just fine.

**Reply by wayland** [reply]
Hmm.  My apologies for not being very clear.  Everything works fine until I run the ERC, and then it complains.  
- I'm using Kicad 6.0.7
- I'm working off OpenDeck r3.1.0

Thanks!

**Comment by wayland** [comment]
OK, I think I've found the problem.  Here are the real errors.  

The following occur when I try to update the PCB from the schematic.  Hope this helps.  

Error: Cannot add U14 (footprint 'igor_footprint_lib:RFM-0505' not found).
Error: Cannot add J12 (footprint 'igor_footprint_lib:TRAPC3MX' not found).
Error: Cannot add J9 (footprint 'igor_footprint_lib:DIN5' not found).
Error: Cannot add J8 (footprint 'Igor:TYPE-C-31-M-12' not found).
